runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/Process/Process.state.ts

```diff
@@ -64,6 +65,12 @@ export interface UpdateDiagramRequest {
     executionInfo?: string | null;
 }
 
+export interface CreateProcessCredentialDto {
+    credentialId: string;
+    templateName: string;
+    processId: string;
+}
```

Comment:
If it has common fields with ProcessCredential type defined in rb-common it could extend/Omit/Pick properties from ProcessCredential

Reply:
done